### PR TITLE
Feature/issue 4 driver version

### PR DESCRIPTION
**Pull request by danielrohe**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -42,6 +42,7 @@
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<spring-boot-etcd.version>1.4</spring-boot-etcd.version>
+		<cassandra-driver.version>3.0.0</cassandra-driver.version>
```
Comment (MALPI) [inline pom.xml]: datastax.driver is more fitting I think.

File: pom.xml [hunk 2]
```diff
@@ -207,4 +262,14 @@
 		</repository>
 	</distributionManagement>
 
+	<profiles>
+		<profile>
+			<id>cassandra-driver-3</id>
```
Comment (MALPI) [inline pom.xml]: same here

File: pom.xml (hunk 1 above)
Comment (danielrohe) [inline pom.xml]: the property is given by spring-boot

**Comment by MALPI** [comment]
lgtm

**Pull request merged**